src/features/profile/pages/ChangeEmail/__tests__/ChangeEmail.native.test.tsx

```diff
@@ -77,4 +77,35 @@ describe('<ChangeEmail/>', () => {
       })
     })
   })
+
+  it('should show error message if the API call is ko', async () => {
+    // @ts-ignore we don't use the other properties of UseMutationResult (such as failureCount)
+    // eslint-disable-next-line local-rules/independant-mocks
+    mockedUseChangeEmailMutation.mockImplementation(({ onError }: UseChangeEmailMutationProps) => ({
+      mutate: () => onError(undefined),
+    }))
+
```

Comment:
Peut-être que tu as déjà essayé, mais si tu gères les cas d'erreurs en dernier dans tes tests, tu peux peut-être le mettre avant le it ? 

```
/* KO:
it('should do something', () => {
    myFunction.mockImplementation(() => {})
    // my test
  })
*/

/* OK:
myFunction.mockImplementation(() => {})
it('should do something', () => {
  // my test
})
*/